compilation error in windows

When compiling opnemeeg as a conda package, we get the [following error](
https://ci.appveyor.com/project/conda-forge/staged-recipes/build/1.0.25380#L2642):
```sh
error C2335: 'timeval': a type cannot be introduced in a function parameter list
```

I manage to replicate it, localy.
```sh
Scanning dependencies of target _openmeeg
[ 98%] Building CXX object wrapping/src/CMakeFiles/_openmeeg.dir/openmeegPYTHON_wrap.cxx.obj
openmeegPYTHON_wrap.cxx
c:\conda\conda-bld\openmeeg_1528727691003\_h_env\include\pytime.h(122): error C2335: 'timeval': a type cannot be introduced in a function parameter list
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEG\include\vect3.h(44): warning C4005: '_CRT_SECURE_NO_DEPRECATE': macro redefinition
C:\conda\conda-bld\openmeeg_1528727691003\work\wrapping\src\openmeegPYTHON_wrap.cxx(145): note: see previous definition of '_CRT_SECURE_NO_DEPRECATE'
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\Exceptions.H(21): warning C4275: non dll-interface class 'std::exception' used as base for dll-interface class 'OpenMEEG::maths::Exception'
C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\INCLUDE\vcruntime_exception.h(43): note: see declaration of 'std::exception'
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\Exceptions.H(21): note: see declaration of 'OpenMEEG::maths::Exception'
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(69): warning C4396: 'io_utils::operator >>': the inline specifier cannot be used when a friend declaration refers to a specialization of a function template
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(165): note: see reference to class template instantiation 'io_utils::imanip<const unsigned char>' being compiled
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(84): warning C4396: 'io_utils::operator >>': the inline specifier cannot be used when a friend declaration refers to a specialization of a function template
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(306): note: see reference to class template instantiation 'io_utils::imanip2<const char *,bool &>' being compiled
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(100): warning C4396: 'io_utils::operator >>': the inline specifier cannot be used when a friend declaration refers to a specialization of a function template
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEGMaths\include\IOUtils.H(505): note: see reference to class template instantiation 'io_utils::imanip3<std::string &,const char,const bool>' being compiled
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEG\include\geometry_io.h(65): warning C4273: 'OpenMEEG::Geometry::load_vtp': inconsistent dll linkage
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEG\include\geometry.h(126): note: see previous definition of 'load_vtp'
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEG\include\geometry_io.h(161): warning C4273: 'OpenMEEG::Geometry::write_vtp': inconsistent dll linkage
C:\conda\conda-bld\openmeeg_1528727691003\work\OpenMEEG\include\geometry.h(127): note: see previous definition of 'write_vtp'
NMAKE : fatal error U1077: 'C:\PROGRA~2\MICROS~1.0\VC\bin\amd64\cl.exe' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.
NMAKE : fatal error U1077: '"C:\Program Files (x86)\Microsoft Visual Studio 14.0\VC\BIN\amd64\nmake.exe"' : return code '0x2'
Stop.

(C:\conda\conda-bld\openmeeg_1528727691003\_h_env) C:\conda\conda-bld\openmeeg_1528727691003\work>if errorlevel 1 exit 1
Traceback (most recent call last):
  File "C:\conda\Scripts\conda-build-script.py", line 10, in <module>
    sys.exit(main())
  File "C:\conda\lib\site-packages\conda_build\cli\main_build.py", line 420, in main
    execute(sys.argv[1:])
  File "C:\conda\lib\site-packages\conda_build\cli\main_build.py", line 411, in execute
    verify=args.verify)
  File "C:\conda\lib\site-packages\conda_build\api.py", line 200, in build
    notest=notest, need_source_download=need_source_download, variants=variants)
  File "C:\conda\lib\site-packages\conda_build\build.py", line 2168, in build_tree
    notest=notest,
  File "C:\conda\lib\site-packages\conda_build\build.py", line 1376, in build
    windows.build(m, build_file, stats=build_stats)
  File "C:\conda\lib\site-packages\conda_build\windows.py", line 297, in build
    check_call_env(cmd, cwd=src_dir, stats=stats)
  File "C:\conda\lib\site-packages\conda_build\utils.py", line 300, in check_call_env
    return _func_defaulting_env_to_os_environ('call', *popenargs, **kwargs)
  File "C:\conda\lib\site-packages\conda_build\utils.py", line 280, in _func_defaulting_env_to_os_environ
    raise subprocess.CalledProcessError(proc.returncode, _args)
subprocess.CalledProcessError: Command '['cmd.exe', '/c', 'bld.bat']' returned non-zero exit status 1.

(build-openmeeg) F:\random-thoughts\staged-recipes\recipes\openmeeg>
```

--- a/OpenMEEG/src/geometry.cpp
+++ b/OpenMEEG/src/geometry.cpp
@@ -40,6 +40,7 @@ knowledge of the CeCILL-B license and that you accept its terms.
 #include <geometry.h>
 #include <geometry_reader.h>
 #include <geometry_io.h>
+#include <ciso646>
 
 namespace OpenMEEG {
 

--- a/OpenMEEG/src/interface.cpp
+++ b/OpenMEEG/src/interface.cpp
@@ -39,6 +39,7 @@ knowledge of the CeCILL-B license and that you accept its terms.
 
 #include <interface.h>
 #include <algorithm>
+#include <ciso646>
 
 namespace OpenMEEG {
 

--- a/OpenMEEG/src/sensors.cpp
+++ b/OpenMEEG/src/sensors.cpp
@@ -40,6 +40,7 @@ knowledge of the CeCILL-B license and that you accept its terms.
 #include <algorithm>
 #include <sensors.h>
 #include <danielsson.h>
+#include <ciso646>
 
 namespace OpenMEEG {
 

--- a/cmake/CompilerOptions.cmake
+++ b/cmake/CompilerOptions.cmake
@@ -1,6 +1,6 @@
 if(MSVC)
     # to allow the use of and, or instead of && ||
-    add_compile_options("/Za")
+    # add_compile_options("/Za")
 endif()
 
 #---------------------------------------------------------------

--- a/cmake/CompilerOptions.cmake
+++ b/cmake/CompilerOptions.cmake
@@ -1,8 +1,3 @@
-if(MSVC)
-    # to allow the use of and, or instead of && ||
-    # add_compile_options("/Za")
-endif()
-
 #---------------------------------------------------------------
 # Test and setup the C++ compiler features
 #---------------------------------------------------------------

